src/premium.cpp

```diff
@@ -115,7 +124,10 @@ class premium_account : public ItemScript
             player->ADD_GOSSIP_ITEM(GOSSIP_ICON_TRAINER, GOSSIP_TEXT_TRAIN, GOSSIP_SENDER_MAIN, GOSSIP_ACTION_INFO_DEF + 8);
```

Comment:
Please use the following format for all gossip functions:
https://github.com/azerothcore/azerothcore-wotlk/blob/dc14094e28ea01448564c482dc9f63ca6b5ed2cf/src/server/game/AI/ScriptedAI/ScriptedGossip.h#L73

"AddGossipItemFor"
"SendGossipMenuFor"
And so on.

Reply:
oof , i never planned to update the whole module like that , i was just adding a new feature , i dont even know how to do that ill have to look at another module , that will take me some time , im kind of a slow learner takes me a lot of time and repetition. maybe i can fully redo this module next year , this is how it is on azerothcore atm

Reply:
Yes indeed you are right, maybe i will fix them myself if i'll have some time

Thanks @milestorme